group.py

```diff
@@ -1,5 +1,17 @@
 """An example of how to represent a group of acquaintances in Python."""
-
+# Testing push/pull
 # Your code to go here...
 
-my_group =
+my_group = {
+    # (name, age) as key
+    # ["job", list of relations as tuples (name, relation to name)] as value
+    ("Jill", 26):["biologist", ("John", "partner"), ("Zalika", "friend")], 
+    ("Zalika", 28):["artist", ("Jill", "friend")],
+    ("John", 27):["writer", ("Jill":"partner")],
+    ("Nash", 34):["cook", ("John", "cousin"), ("Zalika":"landlord")]
+}
```

Comment:
Dictionaries tend to use a single value key such as "Jill".  Here you have put a tuple as a key e.g. the key for the top line is ("Jill", 23).  This would mean that when you reference any data related to Jill you would need both her name and age.  

At the moment referencing Jill would be done using `my_group[("Jill", 26)]` but it might be nicer to make it so that Jill is references like `my_group["Jill"]`